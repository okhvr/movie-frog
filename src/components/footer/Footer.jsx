import React, { Component } from 'react';

import './style.scss';

export default class Footer extends Component {

  render() {
    return (
        <div className="block">
          <section className="bg-section">
            <span>moviefrogsroulette</span>
          </section>
        </div>
    );
  }
}
