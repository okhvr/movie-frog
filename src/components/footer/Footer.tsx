import React, { Component } from 'react';
import styled from 'styled-components';

const StyledWrapper = styled.div`
  background-color: grey;
  position: relative;
  color: white;
  padding: 10px 30px;
`

const StyledSection = styled.section`
  max-width: 1080px;
  margin: auto;
  position: relative;
  z-index: 2;
`
export default class Footer extends Component {

  render() {
    return (
        <StyledWrapper>
          <StyledSection>
            <span>moviefrogsroulette</span>
          </StyledSection>
        </StyledWrapper>
    );
  }
}
