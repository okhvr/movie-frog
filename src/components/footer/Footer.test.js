import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './Footer';

describe('Footer component', () => {
  it('should render correctly', () => {
    const component = renderer.create(
        <Footer/>
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});