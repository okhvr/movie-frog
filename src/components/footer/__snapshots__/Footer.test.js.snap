// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render correctly 1`] = `
<div
  className="block"
>
  <section
    className="bg-section"
  >
    <span>
      moviefrogsroulette
    </span>
  </section>
</div>
`;
